docs/pages/reference/networking.mdx

```diff
@@ -36,21 +36,22 @@ following use cases:
 </TabItem>
 <TabItem scope={["cloud", "team"]} label="Cloud-Hosted">
 
-All Teleport services (e.g., the Application Service and Database Service) have
-an optional `public_addr` property that you can modify in each service's
-configuration file. The public address can take an IP or a DNS name. It can also
-be a list of values:
+For Teleport Enterprise (managed) you choose the sub-domain of
+the domain `teleport.sh` for your account. That fully qualified domain name
+(ex: `example.teleport.sh`) are managed by Teleport for your account 
+along with any sub-domains of it for Teleport Application Service. The public
+address (`public_addr`) for the Teleport Application Service is configurable.
+
+In the case of web applications the public address must be a subdomain of the Teleport account URL
+since the domain and TLS certificates are maintained by Teleport.
```

Comment:
```suggestion
On Teleport Enterprise (Cloud), you can choose the sub-domain of
the domain `teleport.sh` for your account. That fully qualified domain name
(e.g., `example.teleport.sh`) is managed by Teleport for your account 
along with any sub-domains assigned to Teleport-protected applications (e.g., `grafana.example.teleport.sh`).

The public address (`public_addr`) for the Teleport Application Service is configurable. In the case of web applications, the public address must be a subdomain of the Teleport account URL
since the domain and TLS certificates are maintained by Teleport.
```
Grammar and style suggestions